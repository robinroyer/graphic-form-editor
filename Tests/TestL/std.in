LOAD save.txt
LIST
LOAD save02.txt
LIST
EXIT