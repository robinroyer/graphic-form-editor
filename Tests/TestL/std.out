OK
#file successfully loaded
C c01 10 10 4
L pl01 1 2 3 4 5 6 7 8 9 10
R r01 5 6 7 0
OK
#file successfully loaded
C c01 10 10 4
C c02 15 15 3
L pl01 1 2 3 4 5 6 7 8 9 10
R r01 5 6 7 0
R r02 1 3 7 42
